examples/stats-widget/webpack.config.js

```diff
@@ -0,0 +1,43 @@
+// NOTE: To use this example standalone (e.g. outside of repo)
+// delete the local development overrides at the bottom of this file
+
+// avoid destructuring for older Node version support
+const resolve = require('path').resolve;
+
+const BABEL_CONFIG = {
+  presets: ['@babel/env', '@babel/react'],
+  plugins: ['@babel/proposal-class-properties']
```

Comment:
why are we adding this? Not standard, and I don't see it being used in the example...